connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -493,6 +493,7 @@ private WorkerTask buildWorkerTask(ClusterConfigState configState,
         if (task instanceof SourceTask) {
             retryWithToleranceOperator.reporters(sourceTaskReporters(id, connConfig, errorHandlingMetrics));
             TransformationChain<SourceRecord> transformationChain = new TransformationChain<>(connConfig.<SourceRecord>transformations(), retryWithToleranceOperator);
+            log.debug("Initializing: {}", transformationChain);
```

Comment:
I'm actually wondering if this should be INFO level, just so that it's clear what SMTs are actually being used for this connector, in case the user messed up the configuration. Thoughts? Is this logged anywhere else?

Reply:
agreed